flax/linen/module.py

```diff
@@ -541,10 +553,10 @@ def _wrap_module_methods(cls):
       if hasattr(method, 'nowrap'):
         continue
       wrapped_method = wrap_method_once(method)
-      if _use_named_call and key != 'setup':
+      if key != 'setup':
```

Comment:
only downside here is increased stack depth - but we can add stack trace filtering soon.

Reply:
Named call might actually be a context manager soon. So hopefully that will also clean up (and speed up!) tracing